src/lib/utilities/schedule-comment-formatting.ts

```diff
@@ -4,10 +4,10 @@ import { genericWeekDays, monthNames, weekDays } from './calendar';
 
 // Examples of output
 // Every 3hrs:5min:20 sec 30 min offset
-// Weekends at 12:00pm PST
-// Sunday, Monday, Tuesday, Wednesday, Thursday, Friday at 12:00pm UTC
-// Every 1, 15, 31 of the month at 12:00pm UTC
-// Every 1, 15, 31 of January, March, April, June, July, September, December at 12:00pm UTC
+// Weekends at 12:00 PST
+// Sunday, Monday, Tuesday, Wednesday, Thursday, Friday at 12:00 UTC
+// Every 1, 15, 31 of the month at 12:00 UTC
+// Every 1, 15, 31 of January, March, April, June, July, September, December at 12:00 UTC
 
 export const calendarToComment = ({
```

Comment:
Are we even using these utils anywhere? I've updated them, but I don't think they're being used currently.

Reply:
No we aren't since we removed trying to do "human readable" comment labels like Every x days of y ... We can remove it